openlibrary/plugins/openlibrary/js/covers.js

```diff
@@ -112,3 +115,60 @@ export function initCoversSaved() {
         parent.$(cover_selector).attr('src', cover_url);
     }
 }
+
+// This function will be triggered when the user clicks the "Paste" button
+export async function pasteImage() {
+    let formData = null;
+    try {
+        const clipboardItems = await navigator.clipboard.read();
+        for (const item of clipboardItems) {
+            if (!item.types.includes('image/png') && !item.types.includes('image/jpeg') && !item.types.includes('image/jpg')) {
+                continue;
+            }
+
+            const mimeType = item.types.includes('image/png') ? 'image/png' : (item.types.includes('image/jpeg') ? 'image/jpeg' : 'image/jpg');
+            const fileExtension = mimeType === 'image/png' ? 'png' : (mimeType === 'image/jpeg' ? 'jpeg' : 'jpg');
+            const blob = await item.getType(mimeType);
+            const image = document.getElementById('image');
+            image.src = URL.createObjectURL(blob);
+            image.classList.add('visible');
+
+            // Update the global formData with the new image blob
+            formData = new FormData();
+            formData.append('file', blob, `pasted-image.${fileExtension}`);
+
+            // Automatically fill in the hidden file input with the FormData
+            const fileInput = document.getElementById('hiddenFileInput');
+            const file = new File([blob], `pasted-image.${fileExtension}`, { type: mimeType });
+            const dataTransfer = new DataTransfer();
+            dataTransfer.items.add(file);
+            fileInput.files = dataTransfer.files; // This sets the file input with the image
+
+            // Show the upload button and update its text
+            const uploadButton = document.getElementById('uploadButtonPaste');
+            uploadButton.classList.add('visible')
+
+            return formData;
+        }
+        alert('No image found in clipboard');
+    } catch (error) {
+
+    }
+}
+
+export function initPasteForm(coverForm) {
+    const pasteButton = coverForm.querySelector('#pasteButton');
+    let formData = null;
+
+    pasteButton.addEventListener('click', async () => {
+        formData = await pasteImage(coverForm);
```

Comment:
```suggestion
        formData = await pasteImage(coverForm);
        pasteButton.textContent = "Change Image"
```
Update the text of the paste button here.